internal/command/jsonformat/computed/renderers/block.go

```diff
@@ -0,0 +1,122 @@
+package renderers
+
+import (
+	"bytes"
+	"fmt"
+	"sort"
+
+	"github.com/hashicorp/terraform/internal/command/jsonformat/computed"
+
+	"github.com/hashicorp/terraform/internal/command/format"
+	"github.com/hashicorp/terraform/internal/plans"
+)
+
+var (
+	importantAttributes = []string{
+		"id",
+	}
+)
+
+func importantAttribute(attr string) bool {
+	for _, attribute := range importantAttributes {
+		if attribute == attr {
+			return true
+		}
+	}
+	return false
+}
+
+func Block(attributes map[string]computed.Diff, blocks map[string][]computed.Diff) computed.DiffRenderer {
+	return &blockRenderer{
+		attributes: attributes,
+		blocks:     blocks,
+	}
+}
+
+type blockRenderer struct {
+	NoWarningsRenderer
+
+	attributes map[string]computed.Diff
+	blocks     map[string][]computed.Diff
+}
+
+func (renderer blockRenderer) RenderHuman(diff computed.Diff, indent int, opts computed.RenderHumanOpts) string {
```

Comment:
When reading this diff I wondered if `RenderHuman` was part of some interface, and I couldn't immediately see which interface. I've found the pattern of "interface assertions" to be helpful for this, along with ensuring that a given type satisfies its intended interface. What I'm referring to is this kind of module-level expression:

```go
var _ Renderer = (*blockRenderer)(nil)
```

(That assumes a pointer receiver; there's probably an equivalent expression for a struct receiver.)

Reply:
Done!